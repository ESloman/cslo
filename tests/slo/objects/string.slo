var string = "Hello, world!";
print(string.upper());
print(string.lower());
print(string.title());

print("alllowercase".upper());

var split = string.split(",");
for (var part in split) {
    print(part);
}

var unstripped = "  Hello, world!        ";
print(unstripped.strip());

print(string.startswith("Hello"));  # true
print(string.startswith("foo"));    # false
print(string.endswith("world!"));   # true
print(string.endswith("foo"));      # false
