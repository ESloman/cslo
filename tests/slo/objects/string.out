HELLO, WORLD!
hello, world!
Hello, World!
ALLLOWERCASE
Hello
 world!
Hello, world!
true
false
true
false
false
false
false
true
true
true
Hello, mum!
3
4
7
